src/Temporalio/Bridge/Runtime.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace Temporalio.Bridge
+{
+    internal class Runtime : SafeHandle
+    {
+        internal readonly unsafe Interop.Runtime* ptr;
+
+        public Runtime() : base(IntPtr.Zero, true)
+        {
+            using (var scope = new Scope()) { }
+            unsafe
+            {
+                var options = new Interop.RuntimeOptions();
+                var res = Interop.Methods.runtime_new(&options);
+                GC.KeepAlive(options);
```

Comment:
It took me a second to figure out what this does.
It's a bit weird that it keeps `options` alive until `KeepAlive` is called.

Reply:
Yeah, the method actually does nothing, this just tells the compiler not to garbage collect this value. Though I probably should just use that new "Scope" feature I made which keeps multiple things alive.